internal/plugins/helm/v1/scaffolds/api.go

```diff
@@ -70,26 +70,44 @@ func (s *apiScaffolder) scaffold() error {
 	if err != nil {
 		return err
 	}
-	r, chrt, err := chartutil.CreateChart(projectDir, s.opts)
+	resourceOptions, chrt, err := chartutil.CreateChart(projectDir, s.opts)
 	if err != nil {
 		return err
 	}
 
+	//todo(camilamacedo86): replace the options by kubernetes-sigs/kubebuilder#1974
+	if err := resourceOptions.Validate(); err != nil {
+		return err
+	}
+
 	// Check that resource doesn't exist
-	if s.config.HasResource(r.GVK()) {
+	if s.config.HasResource(resourceOptions.GVK()) {
 		return errors.New("the API resource already exists")
 	}
+
 	// Check that the provided group can be added to the project
-	if !s.config.MultiGroup && len(s.config.Resources) != 0 && !s.config.HasGroup(r.Group) {
+	if !s.config.IsMultiGroup() && s.config.ResourcesLength() != 0 && !s.config.HasGroup(resourceOptions.Group) {
 		return errors.New("multiple groups are not allowed by default, to enable multi-group set 'multigroup: true' in your PROJECT file")
 	}
 
-	res := r.NewResource(s.config, true)
-	s.config.UpdateResources(res.GVK())
+	resource := resourceOptions.NewResource(s.config)
+
+	resource.Domain = s.config.GetDomain()
+
+	// remove the path since is not a Golang project
+	resource.Path = ""
+
+	// add the resource API
+	resource.API.Namespaced = true
+	resource.API.CRDVersion = s.opts.CRDVersion
+
```

Comment:
No. It is not. 
If we do not add it here the config will be wrong for Ansible/Helm. 
I will add a todo for we remove that after it is handled properly in upstream. Probably, in the PR where we. will able to use rersource.newReource instead of the golang.options.  

c/c @adirio

Reply:
Did you see the link I posted?